src/sdk/pynni/nni/evo_nas_tuner/evo_nas_tuner.py

```diff
@@ -0,0 +1,167 @@
+import copy
```

Comment:
please write UT for this tuner

Reply:
IT is also required for a new tuner.

Reply:
UT & IT are in progress.

Reply:
UT & IT are completed and passed.

Reply:
The IT case took 525 seconds to finish, there is another classic nas case finished in 82 second.  Is it possible to make it faster?

Reply:
I reduced the number of epochs required in IT and it now costs less than a minute to run locally.